components/03-modules/dashboard-list-item/_dashboard-list-item.scss

```diff
@@ -0,0 +1,40 @@
+$dashboard-list-item-margin : 15px !default;
+$dashboard-list-item-padding: 15px !default;
+
+.dashboard-list-item {
```

Comment:
Name is not apparent - is it list for dashboard or list item for dashboard? In module I see list and from the name I'm thinking that you create only styles for list-item. Please correct naming

Reply:
renamed to `wishlist`

Reply:
renamed to `sidebar-nav-list` :)